packages/protocol/contracts/bridge/IBridge.sol

```diff
@@ -8,31 +8,30 @@ pragma solidity ^0.8.20;
 
 /// @title IBridge
 /// @notice The bridge used in conjunction with the {ISignalService}.
-/// @dev Ether is held by Bridges on L1 and by the EtherVault on L2,
-/// not by token vaults.
+/// @dev Ether is held by Bridges on L1 and L2s.
 interface IBridge {
     struct Message {
         // Message ID.
         uint128 id;
-        // Message sender address (auto filled).
+        // The address, EOA or contract, that interacts with this bridge.
         address from;
-        // Source chain ID (auto filled).
-        uint256 srcChainId;
-        // Destination chain ID where the `to` address lives (auto filled).
-        uint256 destChainId;
-        // User address of the bridged asset.
-        address user;
-        // Destination address.
+        // Source chain ID.
+        uint64 srcChainId;
+        // Destination chain ID where the `to` address lives.
+        uint64 destChainId;
+        // The owner of the message.
+        address owner;
```

Comment:
owner is OK, but need to rename the GenerateGenesis.g.sol `line 190` and `215` accordingly.